models/public/text-recognition-resnet-fc/model.yml

```diff
@@ -291,7 +291,8 @@ conversion_to_onnx_args:
 model_optimizer_args:
   - --input_shape=[1,1,32,100]
   - --input=input
-  - --scale_values=input[255]
+  - --mean_values=input[0.5]
```

Comment:
```suggestion
  - --mean_values=input[127.5]
```

Reply:
@morkovka1337 looks like your configuration in internal repo is incorrect

Reply:
@akorobeinikov could you please also reflect origina input/output and normalization parameters in model README.md

Reply:
There is a preprocessing in the original model which could be confusing for the first time. The cause is that `Normalize` first divides input tensor on 255 and then normalizes with mean=0.5 and std=0.5.
https://albumentations.ai/docs/api_reference/augmentations/transforms/#albumentations.augmentations.transforms.Normalize
It should be double checked but it seems to me that simplified equation is:
(x / 255 - 0.5) / 0.5 = x / 127.5 - **1**
Thus, preprocessing should be: 
```
--scale_values=input[127.5]
--mean_values=input[1]
```

But again, this should be double checked.

Reply:
MO and AC use another normalization ops  order
(x - mean) / std

If you checked with previous version of AC config and model reproduces accuracy, then (x / 255 - 0.5) / 0.5 = (x - 127.5) / 127.5

Reply:
Ok then, your version should be correct. I did not check it, it was just theoretical calculation. =)